prow/tide/gerrit.go

```diff
@@ -225,10 +225,6 @@ func (p *GerritProvider) Query() (map[string]CodeReviewCommon, error) {
 						logger.WithField("id", c.ID).Debug("Change not submittable presented in query results.")
 						continue
 					}
-					if !c.Mergeable {
```

Comment:
Can we still keep this check but modify the Mergeable field to be a pointer so that we can check for its absence (and only skip if the field was not provided in the response from Gerrit)?